dot/rpc/modules/author_test.go

```diff
@@ -309,25 +279,45 @@ func TestAuthorModule_PendingExtrinsics(t *testing.T) {
 	}
 	for _, tt := range tests {
 		t.Run(tt.name, func(t *testing.T) {
+			tt.args.res = PendingExtrinsicsResponse{}
 			cm := &AuthorModule{
 				logger:     tt.fields.logger,
 				coreAPI:    tt.fields.coreAPI,
 				txStateAPI: tt.fields.txStateAPI,
 			}
-			if err := cm.PendingExtrinsics(tt.args.r, tt.args.req, tt.args.res); (err != nil) != tt.wantErr {
-				t.Errorf("AuthorModule.PendingExtrinsics() error = %v, wantErr %v", err, tt.wantErr)
-				return
-			}
-			if diff := cmp.Diff(tt.wantRes, *tt.args.res); diff != "" {
-				t.Errorf("unexpected response: %s", diff)
+			err := cm.PendingExtrinsics(tt.args.r, tt.args.req, &tt.args.res)
+			if tt.wantErr {
+				require.Error(t, err)
+				assert.EqualError(t, err, tt.expErr.Error())
```

Comment:
nit then you can also get rid of `wantErr`

```suggestion
			if tt.expErr != nil {
				assert.EqualError(t, err, tt.expErr.Error())
```